docs/installing-viper.rst

```diff
@@ -75,7 +75,7 @@ other development environment set-up.
 
 To create a new virtual environment for Viper run the following commands:
 ::
-    virtualenv -p /usr/local/lib/python3.6/bin/python --no-site-packages ~/viper-venv
+    virtualenv -p /usr/local/lib/python3.6/bin/python3 --no-site-packages ~/viper-venv
```

Comment:
I think this is my bad, I created a symlink `python` pointing to `python3` I think got copied when I showed the original author my install instructions for 16.04.LGTM

Reply:
And i wonder if it's better to add some examples just like the doc of the solidity
[https://solidity.readthedocs.io/en/develop/solidity-by-example.html](example of the solidity)

Reply:
We have examples in a separate section we are working on integrating into the docs

Reply:
cool